### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by nvzqz** [BurntSushi/rust-memchr#44]
It would be nice if there were somewhere that listed changes between different releases. Currently it seems the only way to do that is by comparing tags, but that's not such a great experience. I know it's a lot of work to make a log for 20 releases, but I think it's worth having.

**Comment by BurntSushi** [comment]
@nvzqz No, sorry, I will not maintain one. Like I said, the activity is infrequent enough that I think the commits themselves are a suitable replacement in this case.
